utils/land-patterns.stanza

```diff
@@ -375,6 +505,60 @@ public pcb-landpattern ipc-two-pin-landpattern (part-name:String, pol?:True|Fals
   make-ipc-two-pin-landpattern(part-name, true, pol?)
   ref-label()
 
+; Make a landpattern with pads along the four edges of the landpattern
+public defn make-quad-edge-landpattern (
```

Comment:
Create a unit test. Besides the fact of check the expected behavior of this function, it will also document how it is used.

Reply:
I added a harness test, which is about as far as we can go at the moment. Testing ocdb is different then other libraries (for example, we use autogenerated tests by preprocessing comments).